docs/experiments-msmarco-passage.md

```diff
@@ -322,8 +316,8 @@ It turns out that optimizing for MRR@10 and MAP yields the same settings.
 
 Here's the comparison between the Anserini default and optimized parameters:
 
-| Setting                                         | MRR@10 |    MAP | Recall@1000 |
-|:------------------------------------------------|-------:|-------:|------------:|
+| Setting                                             | MRR@10 |    MAP | Recall@1000 |
```

Comment:
I think you introduced inconsistencies here? Old table seems fine to me?

Reply:
Oops, I clicked on the file and my editor did some formatting automatically. I will fix that.